src/garage/tf/algos/trpo.py

```diff
@@ -58,7 +58,7 @@ def __init__(self,
                  policy,
                  baseline,
                  scope=None,
-                 max_episode_length=500,
+                 max_episode_length=None,
```

Comment:
please remove

Reply:
please see my other comment on this in NPO